autogluon/training/buildspec.yml

```diff
@@ -38,11 +38,11 @@ images:
   BuildAutogluonGPUTrainPy3DockerImage:
     <<: *TRAINING_REPOSITORY
     build: &AUTOGLUON_GPU_TRAINING_PY3 false
-    image_size_baseline: 26726
+    image_size_baseline: 21000
```

Comment:
Great to see the image size reduction. Does that come from the base image?

Reply:
Yes it comes from the base image, it's smaller